packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1023,13 +1035,12 @@ export default class Meeting extends StatelessWebexPlugin {
     return this.fetchMeetingInfo({
       password, captchaCode
     })
-      .then(() => {
-        Metrics.sendBehavioralMetric(
-          BEHAVIORAL_METRICS.VERIFY_PASSWORD_SUCCESS
-        );
+      .then(() =>
+      // Metrics.sendBehavioralMetric(
+      //   BEHAVIORAL_METRICS.VERIFY_PASSWORD_SUCCESS
+      // );
```

Comment:
I guess this code has been moved outside. I am also thinking this code should stay here and not outside. 

Kindly re-visit

Reply:
Added it back.